ci-operator/step-registry/openshift-extended/test/disasterrecovery/openshift-extended-test-disasterrecovery-ref.metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+	"path": "openshift-extended/test/disasterrecovery/openshift-extended-test-disasterrecovery-ref.yaml",
+	"owners": {
+		"approvers": [
+			"jianzhangbjz",
+			"kuiwang02"
+		],
+		"reviewers": [
```

Comment:
@geliu2016 Please add your name and me in reviewers list. Changes need us to know.

Reply:
Updated.